src/radio/index.js

```diff
@@ -151,6 +151,10 @@ class Radio extends EventEmitter {
     const total = this.order.length;
     const needed = Math.ceil(ratio * total);
     this.emit('skips', this.skips, needed);
+    if (this.skips.size >= needed) {
+      this.getNext();
+    }
+    return true;
```

Comment:
is this return value being used anywhere?

Reply:
🐇